auction9-backend/serverless.yml

```diff
@@ -74,3 +74,12 @@ functions:
           path: myauctions/{id}/stop
           method: put
           cors: true
+
+  # return all my won auctions
+  getMyWonAuctions:
```

Comment:
maybe it will be better to call this lambda getUserWonAuctions. Same goes for method name in handler file.

Reply:
Okay lambda renamed